pkg/streamingpromql/operators/functions/native_histograms.go

```diff
@@ -89,51 +89,56 @@ func HistogramFraction(seriesData types.InstantVectorSeriesData, scalarArgsData
 	return data, nil
 }
 
-// HistogramStdDev returns the estimated standard deviation of observations in a native histogram
-// Float values are ignored.
-func HistogramStdDev(seriesData types.InstantVectorSeriesData, _ []types.ScalarData, _ types.QueryTimeRange, memoryConsumptionTracker *limiting.MemoryConsumptionTracker) (types.InstantVectorSeriesData, error) {
-	fPoints, err := types.FPointSlicePool.Get(len(seriesData.Histograms), memoryConsumptionTracker)
-	if err != nil {
-		return types.InstantVectorSeriesData{}, err
-	}
+func HistogramStdDevStdVar(isStdDev bool) InstantVectorSeriesFunction {
+	// returns either the standard deviation, or standard variance of a native histogram.
+	// Float values are ignored.
```

Comment:
Is this comment in the right place? Looks like it should be the docstring for `HistogramStdDevStdVar`.